benchmarks/extract/node-tar-fmap-sync.js

```diff
@@ -0,0 +1,19 @@
+const cwd = __dirname + '/cwd'
+const rimraf = require('rimraf')
+rimraf.sync(cwd)
+require('mkdirp').sync(cwd)
+process.on('exit', _ => rimraf.sync(cwd))
+const path = require('path')
+const file = process.argv[2] || path.resolve(__dirname, '../npm.tar')
+const fs = require('fs')
+const { O_CREAT, O_TRUNC, O_WRONLY, UV_FS_O_FILEMAP } = fs.constants
```

Comment:
Minor nit: this should provide a default value of `0`.  `undefined` will boolean-or to `0` anyway, but save the JS interpreter having to change the types around.